.github/workflows/integration-tests-against-emulator.yaml

```diff
@@ -17,10 +17,13 @@ jobs:
 
     services:
       emulator:
-        image: gcr.io/cloud-spanner-emulator/emulator:0.8.0
+        image: gcr.io/cloud-spanner-emulator/emulator:1.1.1
         ports:
           - 9010:9010
           - 9020:9020
+        # needed because numeric support isn't on by default in emulator:1.1.1
+        # note: future versions will default enable this and deprecate the flag
```

Comment:
nit; should we leave a TODO here instead of a node for the ease of finding this in future

Reply:
DONE -- added TODO.